app/views/clinics/_clinic_form.html

```diff
@@ -12,13 +12,34 @@
       </div>
       <div class="info-form-right col-sm-6">
         <%= f.number_field :gestational_limit, autocomplete: 'off' %>
-        <%= f.number_field :costs_9wks, autocomplete: 'off' %>
+        <%= f.number_field :costs_05wks, autocomplete: 'off' %>
```

Comment:
This we can almost definitely do with a range. Should make it a little more maintainable and do future us a favor! Thoughts?